### PR TITLE
add more tidb error merssage

### PR DESCRIPTION
**Pull request by wanliyi**
add more tidb error merssage and increase the key length for the blob/text column that adds the primary key.

### CODE REVIEW

**Comment by mrigger** [comment]
The changes look good, thanks a lot for the contribution! Currently, the format check is failing. Could you please use `mvn formatter:format` and then append the changes to the previous commit(s)?

**Comment by wanliyi** [comment]
> The changes look good, thanks a lot for the contribution! Currently, the format check is failing. Could you please use `mvn formatter:format` and then append the changes to the previous commit(s)?

Sorry I'm not very skilled, I have done what you said

File: src/sqlancer/tidb/gen/TiDBAnalyzeTableGenerator.java [hunk 1]
```diff
@@ -31,12 +31,13 @@ public static SQLQueryAdapter getQuery(TiDBGlobalState globalState) throws SQLEx
...
-                                                                                                   * com/pingcap/tidb/
-                                                                                                   * issues/15993
-                                                                                                   */));
+        return new SQLQueryAdapter(sb.toString(),
+                ExpectedErrors.from("https://github.com/pingcap/tidb/issues/15993",
+                        /* https://github.com/pingcap/tidb/issues/15993 */ "doesn't have a default value",
+                        "Index 'PRIMARY' in field list does not exist in table" /*
+                                                                                 * https://github. com/pingcap/tidb/
+                                                                                 * issues/15993
+                                                                                 */));
```
Comment (oraluben) [inline src/sqlancer/tidb/gen/TiDBAnalyzeTableGenerator.java:40]:
```suggestion
        return new SQLQueryAdapter(sb.toString(),
                ExpectedErrors.from("https://github.com/pingcap/tidb/issues/15993", "doesn't have a default value", "Index 'PRIMARY' in field list does not exist in table"));
```

The suggestion seems to not select the right code block, hope you can get it :)

**Comment by wanliyi** [comment]
I hope it can be successful this time

**Comment by mrigger** [comment]
Thanks again for the contribution! While the PR looks good in general, it's not ideal that you have formatting and checkstyle commits. Would it be possible that you combine those commits? Also see https://github.com/sqlancer/sqlancer/blob/master/CONTRIBUTING.md#commit-history.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -110,11 +110,13 @@ jobs:
         run: mvn -B package -DskipTests=true
       - name: Set up CockroachDB
         run: |
-          wget -qO- https://binaries.cockroachdb.com/cockroach-v20.2.6.linux-amd64.tgz | tar  xvz
-          cd cockroach-v20.2.6.linux-amd64/ && ./cockroach start-single-node --insecure &
+          wget -qO- https://binaries.cockroachdb.com/cockroach-v20.2.8.linux-amd64.tgz | tar  xvz
```
Comment (mrigger) [inline .github/workflows/main.yml:113]: I think something went wrong when rewriting the commit history here. Could you take a look at that? Besides this, the PR looks ready to be merged.

Comment (wanliyi) [reply .github/workflows/main.yml]: solved

**Pull request merged**